config.lua

```diff
@@ -74,6 +74,11 @@ add_setting("UI", "background_preview", true, {
     end,
 })
 add_setting("UI", "background_preview_has_text", false, { dependencies = { background_preview = true } })
+add_setting("UI", "in-game_buttons", true, {
+    onchange = function()
+        return not config.get("in-game_buttons")
+    end
+})
```

Comment:
Why are you still putting a onchange function here? It is no longer required as you are getting the value directly already.
```suggestion
add_setting("UI", "in-game_buttons", true)
```